conanfile.py

```diff
@@ -26,33 +26,29 @@ class NovatelEdieConan(ConanFile):
         "shared": False,
         "fPIC": True,
     }
+    implements = ["auto_shared_fpic"]
 
     exports_sources = ["cmake/*", "database/*", "include/*", "src/*", "LICENSE", "!doc", "!test", "CMakelists.txt"]
 
     def set_version(self):
         cmakelists_content = Path(self.recipe_folder, "CMakeLists.txt").read_text()
         self.version = re.search(r"novatel_edie VERSION ([\d.]+)", cmakelists_content).group(1)
 
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
     def configure(self):
         if self.options.shared:
-            self.options.rm_safe("fPIC")
             self.options["spdlog"].shared = True
 
     def layout(self):
         cmake_layout(self)
 
     def requirements(self):
         self.requires("nlohmann_json/[>=3.11 <3.12]", transitive_headers=True)
-        self.requires("spdlog/[>=1.13 <2]", transitive_headers=True, transitive_libs=True)
+        self.requires("spdlog/[>=1.13 <2]", transitive_headers=True, transitive_libs=True, force=True)
```

Comment:
why not just remove this line and allow spdlog-setup to install the version of spdlog that it requires? (1.14.1)

Reply:
It's because conflict resolution for versions is almost non-existent for Conan. The version range effectively selects 1.15 while `spdlog_setup` requires 1.14, and it gives up at that point, declaring an unresolvable version conflict.
So the options are:
1. Use a version specific version that has no conflicts. The versions won't be latest then and a lockfile or a force=True is required to ensure stability against version bumps in upstream recipes.
2. Use a version range with force=True.